package-system/squish-ccr/Dockerfile.x86_64

```diff
@@ -0,0 +1,50 @@
+
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+
+FROM amd64/ubuntu:20.04
+ 
+WORKDIR /data/workspace
+
+ARG DOCKER_BUILD_SCRIPT
+
+# Initilize apt cache
+RUN apt-get clean && apt-get update
+
+# Setup time zone and locale data (necessary for SSL and HTTPS packages)
+RUN DEBIAN_FRONTEND="noninteractive" apt-get -y install tzdata locales keyboard-configuration wget gpg
+
+RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
+    dpkg-reconfigure --frontend=noninteractive locales && \
+    update-locale LANG=en_US.UTF-8
+
+ENV LANG=en_US.UTF-8
+
+# Since we are using ubuntu 20.04, manually configure to get a newer version of cmake than the one that is the default for 20.04
+RUN wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | gpg --dearmor - | tee /usr/share/keyrings/kitware-archive-keyring.gpg >/dev/null && \
+    echo 'deb [signed-by=/usr/share/keyrings/kitware-archive-keyring.gpg] https://apt.kitware.com/ubuntu/ focal main' | tee /etc/apt/sources.list.d/kitware.list >/dev/null && \
```

Comment:
I don't think this is necessary, because this step was only needed for squish-ccr because ubuntu 20.04's default cmake was not compatible (i think). I think I will try and test it out without an updated cmake, which would eliminate the need for this block entirely.

Reply:
Turns out this block (and others) wasn't necessary. The min version of cmake was arbitrarily set in this repo's version of CMakeList.txt, which may be a copy from O3DE's min version at the time. That version was changed to match ubuntu 20.04's default version instead.